tests/test_basicEventScheduler.xml

```diff
@@ -0,0 +1,119 @@
+<Simulation verbosity="debug">
```

Comment:
no gold files for this test

Reply:
That's weird, i wonder why the regression test was green

Reply:
see issue #38